playground/MigrateBench/MigrateRequest.cs

```diff
@@ -6,6 +6,19 @@
 
 namespace MigrateBench
 {
+    struct Endpoint
+    {
+        public string Address;
+        public int Port;
+
+        public Endpoint(string endpoint)
```

Comment:
Should we handle illegal input?

Reply:
Good point. This is test code, but we can use IPEndpoint instead of custom Endpoint.
Pushed a fix for it.